setup.py

```diff
@@ -30,5 +30,6 @@
     keywords='aws athena presto cli',
     classifiers=[
         'Topic :: Utilities'
-    ]
+    ],
+    python_requires='>=3.5'
```

Comment:
The tests are failing because they are being run against Python 2.7 and 3.4 but this `python_requires` restricts installation to Python 3.5+.

You need to update the `.travis.yml` file like so ...
```
python:
  - "3.5"
  - "3.6"
```

Reply:
I checked in a fix